src/scripts/main.js

```diff
@@ -22,6 +22,23 @@ const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
   // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    if (data.hasOwnProperty(key)) {
+      const li = document.createElement('li');
+
+      li.textContent = key;
+
+      if (Object.keys(data[key]).length > 0) {
+        createTree(li, data[key]);
+      }
```

Comment:
Code Quality Issue: You are checking if the object has any keys before making the recursive call. This is unnecessary as the `for...in` loop (or `Object.keys().forEach()`) will simply not execute if the object has no properties. You can remove this check to simplify your code.